flow/platforms/asap7/setRC.tcl

```diff
@@ -1,13 +1,14 @@
-# Liberty units are fF,kOhm
-set_layer_rc -layer M1 -capacitance 1.1368e-01 -resistance 1.3889e-01
-set_layer_rc -layer M2 -capacitance 1.3426e-01 -resistance 2.4222e-02
-set_layer_rc -layer M3 -capacitance 1.2918e-01 -resistance 2.4222e-02
-set_layer_rc -layer M4 -capacitance 1.1396e-01 -resistance 1.6778e-02
-set_layer_rc -layer M5 -capacitance 1.3323e-01 -resistance 1.4677e-02
-set_layer_rc -layer M6 -capacitance 1.1575e-01 -resistance 1.0371e-02
-set_layer_rc -layer M7 -capacitance 1.3293e-01 -resistance 9.6720e-03
-set_layer_rc -layer M8 -capacitance 1.1822e-01 -resistance 7.4310e-03
-set_layer_rc -layer M9 -capacitance 1.3497e-01 -resistance 6.8740e-03
+# correlation result (aes, cva6, ibex, riscv32i)
+# M1 capacitance fixed up from -4.8e-02 to 1e-10 as a minuscule positive value
+set_layer_rc -layer M1 -resistance 7.04175E-02 -capacitance 1e-10
+set_layer_rc -layer M2 -resistance 4.62311E-02 -capacitance 1.84542E-01
+set_layer_rc -layer M3 -resistance 3.63251E-02 -capacitance 1.53955E-01
+set_layer_rc -layer M4 -resistance 2.03083E-02 -capacitance 1.89434E-01
+set_layer_rc -layer M5 -resistance 1.93005E-02 -capacitance 1.71593E-01
+set_layer_rc -layer M6 -resistance 1.18619E-02 -capacitance 1.76146E-01
+set_layer_rc -layer M7 -resistance 1.25311E-02 -capacitance 1.47030E-01
+set_wire_rc -signal -resistance 3.23151E-02 -capacitance 1.73323E-01
+set_wire_rc -clock -resistance 5.13971E-02 -capacitance 1.44549E-01
```

Comment:
This seems odd as the clock is routed on M4-M7 which signals are on M2-M7.  I would expect clocks to see less resistance.

Reply:
Clocks might be seeing more via resistance. I double-checked clocks and signals aren't swapped.